### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by mrcbq**
Hello, In this opportunity I did the following:

- [x] Implement the design from the sneak peek wireframes.
    - Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you would be using Devise in coming milestone)

**Note: Analyze the wireframes deeply and try to identify all the necessary elements and how a user would navigate from one page to another**
    
All Users page wireframe Single User page wireframe All Posts by a given User page wireframe Single Post page wireframe

- [x] Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,16 @@
-<h1>Here is the description and posts for a given user</h1>
+<%= render "shared/user_box", user: @user %>
+<div class = "bio-box-container">
+  <div class = "bio-box">
+    <p>Bio</p>
+    <br>
+    <p><%=@user.bio%></p>
+  </div>
+</div>
+<% @posts.limit(3).each do |post| %>
```
Comment (FrederickMih) [inline app/views/users/show.html.erb]: - [x] Hi @mrcbq, so far you have displayed the last three blog posts of your app(Not For A User) :+1: however, in this milestone, we were told to utilized the methods created when processing data in our models(previous milestone). You name this method as `#recent_posts`. This method should render the **user's 3 most recent post**. We are supposed to reused our methods now. Kindly use this method now :+1:

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,12 @@
-<h1>Here is a list of posts for a given user</h1>
+<%= render "shared/user_box", user: @user %>
+<% @posts.each do |post| %>
+  <a class = "post-box" href="<%= user_post_path(@user, post) %>">
+    <div class = "bio-box-container">
+      <%= render "shared/post_box", post: post, user: @user%>
+    </div>
+      <%= render "shared/comment_box", post: post, limit: 5 %>
```
Comment (FrederickMih) [inline app/views/posts/index.html.erb]: - [x] Kindly check comment above :point_up:  and make use of `#recent_comments` to retrieve the 5 most recent comments of a post.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: :raising_hand: @mrcbq, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





## Highlights :green_circle: 

- Codebase is DRY/clean  (Partials usage) :rock: :rocket: 
- Design in line with given wireframes :rocket: :+1: 


## Required Changes ♻️
**N/A**
_Check the comments under the review._

## Optional suggestions
**N/A**
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (mrcbq) [reply app/views/users/show.html.erb]: Thank you for your suggestion, It's Done!

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (mrcbq) [reply app/views/posts/index.html.erb]: Thank you for your suggestion, It's Done!

Comment (peterrobert) [reply app/views/posts/index.html.erb]: This change has been done well

File: app/views/users/show.html.erb (hunk 1 above)
Comment (peterrobert) [reply app/views/users/show.html.erb]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @mrcbq , 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️
[ ] Great work making all the changes that are requested by the previous reviewer


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**